docs/program/extend/modular-resources/_index.md

```diff
@@ -1,150 +1,528 @@
+
+Two key concepts exist across all Viam resources (both built-in and modular) to facilitate this: [*APIs*](#apis) and [*models*](#models).
 
-Two key concepts exist across all Viam resources (built-in and modular) to facilitate this: *APIs* and *models*
+## Key concepts
 
 ### APIs
 
-Every Viam component and service type exposes an *API*.
-This API describes the interface for the particular component or service type.
-For example, the API of built-in component type [camera](/components/camera) exposes methods such as *GetImage*, and the API of built-in service type [vision](/services/vision) exposes methods such as *GetDetectionsFromCamera*.
+Every Viam [resource](/appendix/glossary/#term-resource) exposes an [Application Programming Interface (API)](https://en.wikipedia.org/wiki/API).
+This can also be understood as the protocol that the resource "speaks".
+Each API is described through <a href="https://developers.google.com/protocol-buffers" target="_blank">protocol buffers</a>.
+Viam SDKs [expose these APIs](/internals/robot-to-robot-comms/).
 
-Each API corresponds with a Viam resource and is described through <a href="https://developers.google.com/protocol-buffers" target="_blank">protocol buffers</a>.
-You can see built-in Viam resource APIs in the <a href="https://github.com/viamrobotics/api" target="_blank">Viam API GitHub repository</a>.
+Each Viam resource's API is uniquely namespaced as a colon-delimited-triplet in the form of `namespace:type:subtype`.
 
-[Viam SDKs](/program/sdk-as-client/) that expose these APIs are available in various programming languages and allow you to control your robots [securely from anywhere](/internals/robot-to-robot-comms/).
+For example:
 
-Viam APIs are uniquely namespaced, with each resource type represented as a *colon-delimited-triplet*.
-For example, the built-in component type *camera*'s API is __rdk:component:camera__.
-The built-in service type *vision*'s API is __rdk:service:vision__
+- The API of built-in component [camera](/components/camera) is `rdk:component:camera`, which exposes methods such as `GetImage()`.
+- The API of built-in service [vision](/services/vision) is `rdk:service:vision`, which exposes methods such as `GetDetectionsFromCamera()`.
+
+{{% alert title="Note" color="note" %}}
+You can see built-in Viam resource APIs in the <a href="https://github.com/viamrobotics/api" target="_blank">Viam GitHub</a>.
+{{% /alert %}}
 
 ### Models
 
-A model is an implementation of a resource type that implements all or some of the API methods of a resource type API.
-Models allow you to control any number of versions of a given resource with a consistent interface.
-This is powerful, because you only need to learn and code against one interface which remains the same for different models of the same component type.
```

Comment:
That last sentence is important to call out somewhere

Reply:
It is called out on line 46! `Models allow you to control different versions of resource types with a consistent interface.`
and the examples following